### PR TITLE
Handle image_embeds in ViltModel

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Handle `image_embeds` in `ViltModel` / `ViltForImagesAndTextClassification`.

(looks like `Vilt` is the first model introducing `image_embeds` argument.)

## More Info

The `image_embeds` in `ViltForImagesAndTextClassification` should have `num_images` dimension as `pixel_values` has, as far as I understand.

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
LGTM, but let's wait for @NielsRogge final review before merging!

File: src/transformers/models/vilt/modeling_vilt.py
Comment (NielsRogge) [inline src/transformers/models/vilt/modeling_vilt.py:821]: Maybe for consistency, we can call the other one `text_batch_size` (instead of `batch_size`).

**Review by NielsRogge — APPROVED** [review]
Thanks for improving this!

Out of interest: were you experimenting with ViLT?

**Comment by ydshieh** [comment]
> Thanks for improving this!
> 
> Out of interest: were you experimenting with ViLT?

Not for this PR. I tried to fix a CI (vit-mae), which was about `test_torchscript`.
It turns out to be related model main input -> I worked/improved on it -> more models involved including ViLT -> I just took this chance to work on this PR (otherwise I would forget it very quickly)

File: src/transformers/models/vilt/modeling_vilt.py
Comment (ydshieh) [reply src/transformers/models/vilt/modeling_vilt.py]: done :-) & wait for green CI

**Pull request merged**
